test/bundling/scripts/useBuildFromSource.js

```diff
@@ -65,15 +65,15 @@ async function run(context) {
   const workspaceRoot = new URL('../../../', import.meta.url);
   await Promise.all(
     [
-      'core',
-      'icons',
+      'material',
+      'icons-material',
       'lab',
       'private-theming',
       'styled-engine',
       'styles',
       'system',
       'types',
-      'unstyled',
+      'core',
```

Comment:
should I revert this?

Reply:
Why revert? Looks like a fix to me 🤷

Reply:
it is not quite related to docs, that's all.

Reply:
I would keep it and rename the PR title to "[core] Fix regressions from directory renaming" or something similar :)